### DISCUSSION TITLE
icons or links are not displayed.

### DISCUSSION THREAD

**Discussion by adumith** [microsoft-search/pnp-modern-search#3565] [General]
Hi guys, 

I have placed the following code in the search result. It is supposed to show me an icon to send an email, chat or call the user but none of the icons are showing.

`<a href="callto:{{getUserEmail (slot item @root.slots.UserEmail)}}"><pnp-icon date-name="Phone" aria-hidden="true"></a>
<a href="https://teams.microsoft.com/l/chat/0/0?users={{getUserEmail (slot item @root.slots.UserEmail)}}"><pnp-icon date-name="Message" aria-hidden="true"></pnp-icon></a>
<a href="mailto:{{getUserEmail (slot item @root.slots.UserEmail)}}"><pnp-icon date-name="Mail" aria-hidden="true"></pnp-icon></a>
`
Any ideas?

Thank you,

**Comment by kasperbolarsen** [comment]
Hi 
I guess a typo has sneaked it way into the pnp-icon part:<pnp-icon dat**a**-name="Phone" aria-hidden="true"></a>

**Reply by adumith** [reply]
Oh boy! You right :)

I was wondering if we can add more fields like for example show languages spoken by employees?

Thank you so much....

**Reply by kasperbolarsen** [reply]
Something like this perhaps? https://www.m365thinking.com/post/flags-in-the-pnp-modern-search-filter
